homeassistant/components/sensor/mitemp_bt.py

```diff
@@ -37,8 +37,8 @@
 
 # Sensor types are defined like: Name, units
 SENSOR_TYPES = {
-    'temperature': ['Temperature', '°C'],
-    'humidity': ['Humidity', '%'],
+    'temperature': [DEVICE_CLASS_TEMPERATURE, 'Temperature', '°C'],
+    'humidity': [DEVICE_CLASS_HUMIDITY, 'Humidity', '%'],
     'battery': ['Battery', '%'],
```

Comment:
This also needs a device class. Otherwise the dict structure will not be uniform, since the length of the list values will differ.

Reply:
It's my fault to forget battery info, I've already added it to new commit.